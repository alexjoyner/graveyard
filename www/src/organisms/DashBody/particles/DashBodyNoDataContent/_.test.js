import React from 'react';
import { shallow } from 'enzyme';
import { DashBodyNoDataContent } from '.';

describe('GagugeBlock NO Data Content', () => {
  let component;
  beforeEach(() => {
    component = shallow(<DashBodyNoDataContent />);
  });
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
