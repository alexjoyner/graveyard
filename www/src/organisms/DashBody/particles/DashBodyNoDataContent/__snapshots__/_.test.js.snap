// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GagugeBlock NO Data Content should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DashBodyNoDataContent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
        <UNDEFINED
          height="250px"
          width="25%"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "height": "250px",
          "width": "25%",
        },
        "ref": null,
        "rendered": null,
        "type": undefined,
      },
    ],
    "type": undefined,
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
          <UNDEFINED
            height="250px"
            width="25%"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "height": "250px",
            "width": "25%",
          },
          "ref": null,
          "rendered": null,
          "type": undefined,
        },
      ],
      "type": undefined,
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
